app/src/main/java/com/sdp/movemeet/Activity/ActivityDescriptionActivity.java

```diff
@@ -141,12 +154,36 @@ public void logout(View view) {
         }
     }
 
+    /*
+    private void getUserName() {
+        DocumentReference docRef = fStore.collection("users").document(userId);
+        docRef.get().addOnCompleteListener(new OnCompleteListener<DocumentSnapshot>() {
+            @SuppressLint("LongLogTag")
+            @Override
+            public void onComplete(@NonNull Task<DocumentSnapshot> task) {
+                if (task.isSuccessful()) {
+                    DocumentSnapshot document = task.getResult();
+                    if (document.exists()) {
+                        Log.d(TAG, "DocumentSnapshot data: " + document.getData());
+                        fullNameString = (String) document.getData().get("fullName");
+                        Log.i(TAG, "fullNameString: " + fullNameString);
```

Comment:
Yes, normally, by moving "createOrganizerView();" right after this line, you should be able to uncomment this block of code and retrieve the organiser ID without any issue 🤞

Reply:
But it's still doesn't work

Reply:
Ok ok we will see again during next sprint how we can resolve this then!

Reply:
Yes I have create a task for it !